Replace "server" with "proxy server" for clarity (#31888)
mdn/content: files/en-us/web/http/methods/connect/index.md

For example, the `CONNECT` method can be used to access websites that use
{{Glossary("TLS")}} ({{Glossary("HTTPS")}}). The client asks an HTTP {{Glossary("Proxy
  server")}} to tunnel the [TCP](/en-US/docs/Glossary/TCP) connection to

> the desired destination. The proxy server then proceeds to make the connection on behalf of
> the client. Once the connection has been established by the proxy server, the
> {{Glossary("Proxy server")}} continues to proxy the TCP stream to and
> from the client.

Reviewer: - The second link to "Proxy server" is not needed; there is one already in the paragraph
- "The proxy server then proceeds..." mentions that it is the proxy server making the connection. We can remove the repeated occurrence of "proxy server" in the next sentence. 
```suggestion
the desired destination. The proxy server then proceeds to make the connection on behalf of
the client. Once the connection is established, the
proxy server continues to relay the TCP stream to and
from the client.
```
